### DISCUSSION TITLE
Is GrapesJS Studio (New Version) available for use?

### DISCUSSION THREAD

**Discussion by alexgarciab** [GrapesJS/grapesjs#5849] [Q&A]
I've checked out the new version of GrapesJS (https://studio.grapesjs.com/) and it looks amazing. However, I was checking the documentation and all refers to the old version of GrapesJS. I was wondering if the GrapesJS Studio version is available for use already or if it's not yet available (due to being in alpha version). Thanks

**Comment by artf** [comment]
It's not a new version, it's an application built on top of the GrapesJS framework.

**Reply by alexgarciab** [reply]
Ok, I see. So then, how can I replicate that GrapesJS Studio look?

**Reply by artf** [reply]
The Studio is built with custom UI (on top of https://github.com/GrapesJS/react). The [documentation](https://grapesjs.com/docs/) shows customization steps for every module.

**Reply by alexgarciab** [reply]
I understand that. But how can I replicate that custom UI? Is there a config file available that I can import to the base GrapesJS structure to get that new UI of GrapesJS Studio?

**Reply by alexgarciab** [reply]
@artf any updates? is there such a config template available?

**Reply by artf** [reply]
You can build your own from scratch, the Studio app code is not public.

**Reply by gh-darvishani** [reply]
@artf  is there any possibility to make it public!?
honestly, Studio is awesome but grapesjs ... you know
